### ISSUE TITLE
Easy edit of opening_hours

### ISSUE THREAD

**Issue by Filip10** [Helium314/StreetComplete#695]
**Use case**

If you want to check/change opening hours of shops or etc., nowadays you need to have task for it or edit it straight in tag edit, which is a bit hard.

**Proposed Solution**

Make opening hours easier to edit with showing window like "check opening hours" task have.

**Comment by mnalis** [comment]
I agree, I needed that too... More generally, it would be nice to somehow temporarily force ***any quest*** to be re-asked on specific POI (e.g. by temporarily faking very old `check_date:*`)?

Possible existing workarounds:
- in raw tag editor, delete `opening_hours` - then new quest for `opening_hours` (with nice UI) re-appears below. (disadvantage is that information must be entered from scratch - it cannot be just _updated_ with nice UI)
- in SCEE settings / Quests / Advanced resurvey (for `opening_hours` tag)
- fix `opening_hours` in [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door) instead

**Comment by HolgerJeromin** [comment]
Ref https://github.com/streetcomplete/StreetComplete/issues/4516

**Comment by mnalis** [comment]
> in raw tag editor, delete opening_hours - then new quest for opening_hours (with nice UI) re-appears below. (disadvantage is that information must be entered from scratch - it cannot be just updated with nice UI)

And ***correction*** (it actually _"**can** be just updated with nice UI"_), and thus my new favorite[^1] workaround:
- use raw tag editor to add `check_date:opening_hours=1970-01-01` (gets remembered in MRU cache too, so is only much typing the first time)
- `opening_hours` quest pops up in raw tag editor; open and solve it (all existing `opening_hours` data is present, so you can just edit the changes instead of having to input it all from scratch!)
- (optional) remove updated `check_date:opening_hours` in raw tag editor if not needed (but can be left in too)
- confirm raw tag editor as done 

(Should work for other tags which support resurveys too.)

---

Still, it would be nice if there was **some button which would show all quests that apply to that element again**, even if they were solved recently. Just using generic `check_date` doesn't seem to work, and adding dozes of fake `check_date:*` tags just to show all quests again is too much work. 
(Perhaps setting the phone clock in the future would be quick kludge too, but that is likely to break many things 😺 ). 

Similar idea was suggested (and rejected) in like suggested in https://github.com/streetcomplete/StreetComplete/issues/3918 but SCEE might have better UI (i.e. just extra button in raw tag editor to force re-showing all quests)?


[^1]: as my brain is too used to SC opening_hours editor and can't really get used to EveryDoor opening_hours editor without massive delay.

**Comment by Helium314** [comment]
> Still, it would be nice if there was some button which would show all quests that apply to that element again

You could try changing the element timestamp in [`TagEditor`](https://github.com/Helium314/SCEE/blob/modified/app/src/main/java/de/westnordost/streetcomplete/quests/TagEditor.kt#L145) to something old.
